.circleci/ci/src/jobs/job-release-commit-and-prepare-next-version.ts

```diff
@@ -82,6 +79,10 @@ sed -i 's#version: ".*"#version: "${nextVersion}${nextQualifier}-SNAPSHOT"#' gra
 sed -i 's#"version": ".*"#"version": "${nextVersion}${nextQualifier}-SNAPSHOT"#' gravitee-apim-console-webui/build.json
 sed -i 's#"version": ".*"#"version": "${nextVersion}${nextQualifier}-SNAPSHOT"#' gravitee-apim-portal-webui/build.json
 
+# Helm chart increase version and clean the artifacthub.io/changes annotation
+sed "0,/version.*/s/version.*/version: ${nextVersion}${nextQualifier}/" -i helm/Chart.yaml
+sed '/artifacthub.io\\/changes/q' -i helm/Chart.yaml
```

Comment:
```suggestion
sed -e "0,/^version:/{s/version:.*/version: ${nextVersion}${nextQualifier}/}" -e '/artifacthub.io\/changes/,${ s/|//; /^[ ]*\- /d }' -i helm/Chart.yaml
```
I suggest one sed to do all the things.